client/src/views/Home.vue

```diff
@@ -32,6 +31,8 @@ export default {
     uploading: false,
   }),
   computed: {
+    ...mapState('Filetypes', ['filetypes']),
+    ...mapState('Location', ['location']),
```

Comment:
I don't see why `location` was added, there's already a computed getter and setter below.
```suggestion
```